.changeset/slick-cougars-smoke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@forgerock/davinci-client': minor
```

Comment:
because this affected both packages, we thought it made sense to include both here.